log/log.go

```diff
@@ -77,8 +79,20 @@ type OsqueryLogAdapter struct {
 	kitlog.Logger
 }
 
+var callerRegexp = regexp.MustCompile(`[\w.]+:\d+]`)
+
+func extractOsqueryCaller(msg string) string {
+	if match := callerRegexp.FindString(msg); len(match) > 0 {
+		return strings.TrimSuffix(string(match), "]")
+	} else {
```

Comment:
No need for else if there's a return in the previous branch. 

I think there's a linter that catches this case.

Reply:
We can actually remove the `if` entirely ;) We only ended up here because I was tinkering around with the implementation.